documental/serializers.py

```diff
@@ -70,3 +70,39 @@ def to_representation(self, instance):
         return url_document
 
 
+class DocumentosTISerializers(serializers.ModelSerializer):
+    """Serializer to return action category `models.DocumentalDocs` data.
+
+    Data only for the action category linked to DOCUMENTAL_TI.
+    """
+    usuario_id = UsuarioSerializers()
```

Comment:
Missing spaces.
You are doing the same thing every pull request.

Reply:
ok.